components/molecules/BenefitTasks.js

```diff
@@ -0,0 +1,89 @@
+import PropTypes from 'prop-types'
+
+export default function BenefitTasks(props) {
+  const tasks = [
+    {
+      task: 'Task 1',
+      taskIcon: '/placeholder.svg',
+      taskLink: '/dashboard',
+    },
+    {
+      task: 'Task 2',
+      taskIcon: '/placeholder.svg',
+      taskLink: '/dashboard',
+    },
+    {
+      task: 'Task 3',
+      taskIcon: '/placeholder.svg',
+      taskLink: '/dashboard',
+    },
+    {
+      task: 'Task 4',
+      taskIcon: '/placeholder.svg',
+      taskLink: '/dashboard',
+    },
+    {
+      task: 'Task 5',
+      taskIcon: '/placeholder.svg',
+      taskLink: '/dashboard',
+    },
+    {
+      task: 'Task 6',
+      taskIcon: '/placeholder.svg',
+      taskLink: '/dashboard',
+    },
+    {
+      task: 'Task 7',
+      taskIcon: '/placeholder.svg',
+      taskLink: '/dashboard',
+    },
+    {
+      task: 'Task 8',
+      taskIcon: '/placeholder.svg',
+      taskLink: '/dashboard',
+    },
+    {
+      task: 'Task 9',
+      taskIcon: '/placeholder.svg',
+      taskLink: '/dashboard',
+    },
+  ]
+  let displayedTasks = props.isExpanded ? tasks : tasks.slice(0, 6)
+
+  return (
+    <div className="bg-gray-lighter mt-6 px-8 py-6">
+      <h3 className="font-display font-bold text-xl">
+        {props.isExpanded
```

Comment:
It has been changed in Figma and you can remove condition. "All ... Task" for non expanded.